torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -516,3 +516,309 @@ def forward(self, *inputs: Any) -> Any:
             mix = F.to_image_pil(mix)
 
         return self._unflatten_and_insert_image_or_video(flat_inputs_with_spec, mix)
+
+
+class _AutoAugmentDetectionBase(_AutoAugmentBase):
+    def _flatten_and_extract_image_or_video_and_bboxes(
```

Comment:
I think we have to figure out a generic way to do this things: 
- _flatten_and_extract_image_or_video_and_bboxes
- _unflatten_and_insert_image_or_video_and_bboxes
- _apply_image_or_video_and_bboxes_transform

Reply:
I agree. Currently it’s just a temporary solution. If we can get a generic structure, things would be much better.